CorsixTH/Lua/world.lua

```diff
@@ -534,7 +534,6 @@ function World:spawnVIP(name)
 
   vip.enter_explosions = hospital.num_explosions
 
-  local spawn_point = self.spawn_points[math.random(1, #self.spawn_points)]
```

Comment:
Removing this one is wrong, it's used at the line below
It's also not defined above, it seems

(there was one such line duplicate, but that has been removed already, maybe your patch rebase broke made the wrong guess here?)